### PR TITLE
Expose layer source type and URL in LayerManager

### PR DESCRIPTION
**Pull request by jmacura**
image
---
image
---
image
---
image

### CODE REVIEW

**Comment by jmacura** [comment]
I kindly ask for translations in LV and SK, @DailisLangovskis @FilipLeitner 🙂

**Comment by jmacura** [comment]
resolves #1502

File: projects/hslayers/src/components/utils/layer-utils.service.ts [hunk 1]
```diff
@@ -191,12 +193,58 @@ export class HsLayerUtilsService {
   }
 
   /**
-   * @ngdoc method
-   * @name HsLayerUtilsService#isLayerInManager
-   * @param {Ol.layer} layer Layer to check
-   * @returns {boolean} True if showInLayerManager attribute is set to true
-   * @description Test if layer is shown in layer switcher
+   * Test if the features in the vector layer come from a GeoJSON source
```
Comment (DailisLangovskis) [inline projects/hslayers/src/components/utils/layer-utils.service.ts:196]: I would might try testing if layer is vector layer with utils function and then use switch case to get the format to make code shorter, but I don't know if there is a special need for this format.

**Comment by raitisbe** [comment]
Good job guys! Can I ask @jmacura to squash the style commits into one, to not make the commit log unnecessary long?

**Comment by jmacura** [comment]
> 
> 
> Good job guys! Can I ask @jmacura to squash the style commits into one, to not make the commit log unnecessary long?

Thanks. I will try my best.

**Comment by jmacura** [comment]
> 
> 
> > Good job guys! Can I ask @jmacura to squash the style commits into one, to not make the commit log unnecessary long?
> 
> Thanks. I will try my best.

@raitisbe I have reduced the number of commits to half. Wanted to preserve the original commit author so I kept Filip's and Dailis' commits separate (I don't think it's possible in git to record more then 2 commit authors).

File: projects/hslayers/src/components/utils/layer-utils.service.ts (hunk 1 above)
Comment (jmacura) [reply projects/hslayers/src/components/utils/layer-utils.service.ts:196]: Good point. I generally wanted to keep the same logic of boolean-ish functions. That was the only reason really. Your proposal would be more concise of course.

Comment (jmacura) [reply projects/hslayers/src/components/utils/layer-utils.service.ts:196]: I have refactored. It is not exactly as you meant (if I am correct), but it's at least shorter.

**Comment by jmacura** [comment]
Good to go from my site.

**Review by raitisbe — APPROVED** [review]

**Pull request merged**